packages/components/src/higher-order/with-fallback-styles/index.js

```diff
@@ -45,10 +45,14 @@ export default ( mapNodeToProps ) =>
 						this.nodeRef,
 						this.props
 					);
+
 					if ( ! isEqual( newFallbackStyles, fallbackStyles ) ) {
 						this.setState( {
 							fallbackStyles: newFallbackStyles,
-							grabStylesCompleted: !! every( newFallbackStyles ),
```

Comment:
Noting that both `every` and `Array.prototype.every()` always return a `Boolean`, so the `!!` is not necessary.